lib/hex_web/store/local.ex

```diff
@@ -31,10 +31,19 @@ defmodule HexWeb.Store.Local do
     File.write!(Path.join(dir, "registry.ets.gz"), :zlib.gzip(data))
   end
 
+  def put_registry_signature(signature) do
+    File.mkdir_p!(dir)
+    File.write!(Path.join(dir, "registry.ets.signed"), signature)
```

Comment:
This doesn't make sense, as we're not signing the tarball, but rather the uncompressed ets registry?